plugin/markdown/plugin.js

```diff
@@ -181,18 +190,22 @@ const Plugin = () => {
 
 		// flatten the hierarchical stack, and insert <section data-markdown> tags
 		for( let i = 0, len = sectionStack.length; i < len; i++ ) {
+			let newOptions = {...options}
```

Comment:
what is `new` about the options?

Reply:
newOptions is used to create options for each slide. So that, the options with default metadata won't get changed. Is the wording confusing?

Reply:
yes it is

Reply:
I have renamed it to `slideOptions` and added a brief comment. I hope that explains better.